README.md

```
@@ -12,17 +12,35 @@ Collectors that are enabled by default can be disabled by providing a `--no-coll
 To enable only some specific collector(s), use `--collector.disable-defaults --collector.<name> ...`.
 
 ### Enabled by default
+
+#### Memory Collectors
+Name     | Description
```

markdownlint-cli2 (0.20.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)